business_requirement_deliverable_project/tests/test_br.py

```diff
@@ -0,0 +1,286 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.tests import common
+from openerp.osv import osv
```

Comment:
@moylop260 I would rather keep it since the others modules has passed with this deprecated feature and don't have impact with Travis, Runbot but change for the next version.

@elicoidal @dreispt @seb-elico ?